cfgov/v1/atomic_elements/organisms.py

```diff
@@ -663,18 +663,51 @@ class Meta:
 
 
 class ChartBlock(blocks.StructBlock):
-    element_id = blocks.CharBlock(
+    title = blocks.CharBlock(required=True)
+    # todo: make radio buttons
+    chart_type = blocks.ChoiceBlock(choices=[
+        ('bar', 'Bar'),
+        ('line', 'Line'),
+        ('tile_map', 'Tile Map'),
+    ], required=False)
+    # todo: make radio buttons. required because we need it for the CSS to apply colors to each report page chart
```

Comment:
For report_type field, let's make it more generic and call it color scheme. The options should be green, blue, teal, or navy. Then it can be reused for other projects' charts. We'll update the css to use the new data attribute value too